packages/TorneloScoresheet/src/types/ChessBoardPositions.ts

```diff
@@ -31,6 +31,74 @@ export type Row = '1' | '2' | '3' | '4' | '5' | '6' | '7' | '8';
 export type Column = 'a' | 'b' | 'c' | 'd' | 'e' | 'f' | 'g' | 'h';
 export type Position = `${Column}${Row}`;
 
+// All the squares on a board
+export const BOARD_SQUARES: Position[] = [
```

Comment:
what's this for?

Reply:
Ah yep, this doesn't belong in this PR. Thanks!